.github/workflows/ci.yml

```diff
@@ -10,6 +10,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install packages
-      run: npm install
+      run: yarn install
```

Comment:
doh, thanks for moving us to yarn. My bad on not including on the previous work here.